daemon/inertiad/notifier/notifier.go

```diff
@@ -0,0 +1,55 @@
+package notifier
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+)
+
+// Notifier manages notifications
+type Notifier interface {
+	Notify(string) error
+}
+
+// SlackNotifier represents slack notifications
+type SlackNotifier struct {
+	hookURL string
+}
+
+// NewNotifier creates a notifier with web hook url to slack channel
+func NewNotifier() *SlackNotifier {
+	url := "https://hooks.slack.com/services/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"
```

Comment:
we probably want to make this URL configurable, ie with

```go
func NewNotifier(webhook string) *SlackNotifier
```

also it's fine for now because the Slack workspace is a test one, but we want to avoid committing sensitive information into the repository (although this workspace might be useful for running integration tests, assuming it'll always be available for use)

Reply:
another idea - if a empty string is provided, the notifier should become a no-op notifier (ie it doesn't do anything on `Notify`)